src/project/types/website/website-config.ts

```diff
@@ -103,9 +104,11 @@ export interface OpenGraphConfig {
 
 type WebsiteConfigKey =
   | "title"
+  | "image"
```

Comment:
Be sure to add `image` and `site-description` to the website schema (or just note that it should be done when creating the documentation)

Reply:
Will do as a part of listing schema/doc update.